tests/frontend/test_ratings.py

```diff
@@ -10,6 +10,84 @@
 from pages.desktop.frontend.reviews import Reviews
 
 
+@pytest.mark.serial
+@pytest.mark.nondestructive
+def test_rating_term(selenium, base_url, variables):
+    # this test checks that if the rating has no text, the term used in messages is 'rating'
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    addon.login('rating_user')
+    addon.ratings.rating_stars[3].click()
+    addon.ratings.wait_for_delete_link()
+    assert 'rating' in addon.ratings.delete_rating_link.text
+    addon.ratings.delete_rating_link.click()
+    assert 'rating' in addon.ratings.ratings_card_summary
+    assert 'rating' in addon.ratings.delete_confirm_button.text
+    assert 'rating' in addon.ratings.keep_review_button.text
```

Comment:
Couldn't we integrate this test in `test_rating_without_text` ?

Reply:
Ok